release/rancher/rancher.go

```diff
@@ -98,14 +99,19 @@ func GeneratePrimeArtifactsIndex(path string) error {
 func generateArtifactsIndexContent(listBucket ListBucketResult) ArtifactsIndexContent {
 	indexContent := ArtifactsIndexContent{
 		GA: ArtifactsIndexContentGroup{
-			Versions: map[string][]string{},
-			BaseURL:  rancherArtifactsBaseURL,
+			Versions:      []string{},
+			VersionsFiles: map[string][]string{},
+			BaseURL:       rancherArtifactsBaseURL,
 		},
 		PreRelease: ArtifactsIndexContentGroup{
-			Versions: map[string][]string{},
-			BaseURL:  rancherArtifactsBaseURL,
+			Versions:      []string{},
+			VersionsFiles: map[string][]string{},
+			BaseURL:       rancherArtifactsBaseURL,
 		},
 	}
+	versions := []string{}
```

Comment:
```suggestion
	var versions []string
```

Let's make sure we use the 0 value initialization for variables.